### ISSUE TITLE
Html output not working

### ISSUE THREAD

**Issue by goldilocked** [MicrosoftDocs/visualstudio-docs#2670]
This tutorial is very simple and easy, but even after following these steps I haven't been getting an output link for my coded UI tests. 
I'm using the 2017 enterprise version and went to the config files. I wasn't sure which .NET framework I was using so I first tried one then the other. After each, I closed visual studio, reopened, built the solution, and ran one of my tests. There's no output option for my tests. 
When I hardcode my individual tests using loggerOverrideState I do get the output link. Do you have any advice for what to do next if the instructions here don't work like what happened with me?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: beb72d4a-65af-25d0-82c9-ce448f513139
* Version Independent ID: ff79c27a-44b9-21cf-fbdf-1af82246a3d6
* Content: [Analyzing Coded UI Tests Using Coded UI Test Logs - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/test/analyzing-coded-ui-tests-using-coded-ui-test-logs?view=vs-2017)
* Content Source: [docs/test/analyzing-coded-ui-tests-using-coded-ui-test-logs.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/test/analyzing-coded-ui-tests-using-coded-ui-test-logs.md)
* Product: **visual-studio-windows**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

**Comment by gewarren** [comment]
@boraprachi or @pvlakshm Do you know if these instructions are still valid to enable logging?

Target .NET Framework version 4 with no App.config file present in the test project:

1. Open the QTAgent32_40.exe.config file. By default, this file is located in %ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\Common7\IDE.
2. Modify the value for EqtTraceLevel to the log level you want.
3. Save the file.

Target .NET Framework version 4.5 with no App.config file present in the test project:

1. Open the QTAgent32.exe.config file. By default, this file is located in %ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\Common7\IDE.
2. Modify the value of the EqtTraceLevel to the log level you want.
3. Save the file.

**Comment by gewarren** [comment]
Oops, tagged the wrong Prachi Bora @PBoraMSFT.

**Comment by jayaranigarg** [comment]
@gewarren : Yes, these instructions are still valid. 
You should see which QTAgent*.exe process is getting launched for running the tests and modify the value of EqtTrace in that particular QTAgent*.exe.config file.

**Comment by gewarren** [comment]
@goldilocked Perhaps watch the Details tab in Task Manager to see which QTAgent*.exe process is launched when you run the test. Then, modify the corresponding .config file, as Jayarani suggests. Also, do you have an App.config file in your test project?

![image](https://user-images.githubusercontent.com/24882762/58671047-0e993c80-82f6-11e9-8062-af3f9c8f2817.png)

**Comment by goldilocked** [comment]
I looked at details and saw QTAgent32.exe was running every time I ran a CodedUI test. I went and checked QTAgent32.exe.config and it's one of the files I changed when I first started trying to add automatic tracking. 

I don't believe there's an App.Config for this specific test project. I looked and noticed that another test project for a different type of test has an App.Config, just to cover my bases I tried adding <add name="EqtTraceLevel" value="4" /> in a systems diagnostics node to see if that would affect my tests, it didn't and I'm still getting one png instead of the full action html logging I should be getting. I don't see any other app.config files in the solution. 

What I ended up doing was just adding Playback.PlaybackSettings.LoggerOverrideState = HtmlLoggerState.AllActionSnapshot; as the first line to every test. At this point it's not worth spending the extra time to get it to work automatically when I can copy and paste this line and get what I need. Especially when we're phasing out our CodedUI project soon to prepare for VS 2019

Thank you for your help, I appreciate you taking some time to help me with this issue!
